todx/main.py

```diff
@@ -1,7 +1,6 @@
 #!/usr/bin/env python3
 # -*- coding: utf-8 -*- 
 # import os
-import argparse
```

Comment:
Please also remove this from requirements.txt as well.

Reply:
requirements.txt only has 3 libraries: 
```
appdirs
fuzzywuzzy[speedup]
pytest
```

Which one to remove?